packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -145,7 +145,7 @@ contract Deploy is Deployer {
     /// @notice Deploy the Safe
     function deployL2NativeToken() public onlyDevnet broadcast {
         address addr_ = getL2NativeToken();
-        cfg.setNativeTokenAddress(addr_);
+        cfg.setNativeTokenAddress(addr_, string.concat(vm.projectRoot(), "/deploy-config/", deploymentContext, ".json"));
```

Comment:
This function only works for devnet with the modifier onlyDevnet. I think we maybe should update this function to work with deploymentContext.

Reply:
Thank you!
I forgot to explain!
This function is to change the nativeTokenAddress of devnetl1.json to the actually distributed address. (Currently, the address set in the template goes to devnetL1.json.)
I think this should only work on devnet. 🙇🏼‍♂️

Reply:
Thank you very much!

I understood.